alliance/config/settings/base.py

```diff
@@ -140,8 +160,9 @@ def template_folder_paths(app):
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': PROJECT_APP_TEMPLATES + BASE_TEMPLATES + EXTENSION_TEMPLATES,
-        'APP_DIRS': True,  # TODO
+        #'DIRS': PROJECT_APP_TEMPLATES + BASE_TEMPLATES + EXTENSION_TEMPLATES,
+        'DIRS': [os.path.join(PROJECT_ROOT, 'apps/shared/templates')],
```

Comment:
This would disable each app from having a `templates` directory within them.

E.g.

`apps/backlog/templates/` where **backlog** is the app.

What is the current behavior?